.gitignore

```diff
@@ -140,5 +140,5 @@ dmypy.json
 cython_debug/
 
 .vscode
-
 .DS_Store
+/data
```

Comment:
I'm not sure this will match anything because it's an absolute path, probably should write `./data` or just general `data`